content/en/docs/languages/java/instrumentation.md

```diff
@@ -33,6 +33,13 @@ _natively_ or you can make use of
 
 {{% /alert %}}
 
+## Prerequisites
+
+Ensure that you have the following installed locally:
```

Comment:
Those prerequisites do only apply to the sample app, I would argue we move it there and call it out specifically.

I am wondering if we need a global prerequisites as well, to verify what minimum JDK version is needed overall?

Reply:
ya, I share @jack-berg's https://github.com/open-telemetry/opentelemetry.io/pull/3877#discussion_r1469862533

> someone reading this may infer that we only support Java 17+

and https://github.com/open-telemetry/opentelemetry.io/pull/3877#discussion_r1469879885

> The JDK version compatibility is a pretty important point in the java ecosystem, so while the entire story may be documented elsewhere, I think any spot which mentions version requirement conflicting with java 8+ is prone to being misinterpreted